fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/client/item/v1/AnimationUpdateHandler.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.item.v1;
+
+import net.minecraft.item.ItemStack;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+/**
+ * Provides information on whether an {@link net.minecraft.item.Item} should run its update animation, such as for NBT updates.
+ */
+@Environment(EnvType.CLIENT)
+public interface AnimationUpdateHandler {
+	/**
+	 * Called once per tick to check whether the given held {@link ItemStack} should run the update animation.
```

Comment:
```suggestion
	 * Called once per frame to check whether the given held {@link ItemStack} should run the hand wave update animation.
```

Reply:
As far as I can tell this is actually called per tick and not per frame.

Reply:
My bad then